src/tokenized-line.coffee

```diff
@@ -47,10 +57,15 @@ class TokenizedLine
       tokenStartColumn += token.screenDelta
 
     if token.isAtomic and tokenStartColumn < column
-      if skipAtomicTokens
+      if clipToAtomicTokenEdge == 'right'
         tokenStartColumn + token.screenDelta
-      else
+      else if clipToAtomicTokenEdge == 'left'
         tokenStartColumn
+      else #'closest'
+        if column > tokenStartColumn + token.screenDelta / 2
```

Comment:
Yeah, it's correct, but I should definitely add parentheses to make it more clear. What we're doing here is seeing if we're more than halfway through the token, so we're interested in just dividing the token width by 2.

If you add parentheses the wrong way (e.g., `column > (tokenStartColumn + token.screenDelta) / 2`) then it successfully fails a test:
```
TextEditor
  cursor
    .moveRight()
      it moves the cursor by one column to the right
        Expected { row : 3, column : 5 } to equal [ 3, 4 ].
```